financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkinglinkloginwarmup/NetworkingLinkLoginWarmupScreen.kt

```diff
@@ -74,72 +60,51 @@ internal fun NetworkingLinkLoginWarmupScreen() {
 @Composable
 private fun NetworkingLinkLoginWarmupContent(
     state: NetworkingLinkLoginWarmupState,
-    onCloseClick: () -> Unit,
     onContinueClick: () -> Unit,
-    onCloseFromErrorClick: (Throwable) -> Unit,
     onSkipClicked: () -> Unit,
 ) {
     val lazyListState = rememberLazyListState()
-    FinancialConnectionsScaffold(
-        topBar = {
-            FinancialConnectionsTopAppBar(
-                showBack = true,
-                onCloseClick = onCloseClick
-            )
-        }
+    Box(
```

Comment:
Nit: Can we avoid the nesting?

Reply:
https://github.com/stripe/stripe-android/pull/7923#:~:text=Remove%20wrapping%20box.-,c61e60e,-Merge%20state

Reply:
Actually yes! -> https://github.com/stripe/stripe-android/pull/7923/commits/c61e60ea7d84b475c637fd0ab7ea37e588c85205